src/audio/module_adapter/module_adapter.c

```diff
@@ -847,6 +847,15 @@ int module_adapter_trigger(struct comp_dev *dev, int cmd)
 {
 	comp_dbg(dev, "module_adapter_trigger(): cmd %d", cmd);
 
+	/*
+	 * If the module doesn't support pause, keep it active along with the rest of the
```

Comment:
@ranj063 I wonder why mixer can't support pause?!

Reply:
@dbaluta thats how it was implemented for IPC3. The mixer was always kept running generating silence during pause. As to why I'm not sure. That part of the code was not mine originally so I didnt change the behaviour.